### DISCUSSION TITLE
Enter cursive letters as constants

### DISCUSSION THREAD

**Discussion by kpvier** [Qalculate/qalculate-gtk#518] [Q&A]
Planck's constant is set to be _h_ (italic type) and hours as h (regular roman type). How can I insert the cursive _h_ in an expression such as $E=hf$ without the h referring to hours and giving wrong results?

**Comment by hanna-kn** [comment]
Qalculate uses the Unicode symbol `ℎ` (U+210E) for the Planck constant (and `ℏ` for the reduced variant). Regular h is not supported. I recommend using `planck` for input.

**Reply by kpvier** [reply]
While there seems not to be a way to insert these Unicode symbols, I avoided the problem by creating a new shortcut (`Ctrl+Alt+H`) that inserts variable (with copy-paste *h*). I don't see the value of Unicode characters as abbreviations if one couldn't use them as such.

Thanks for clearing of the symbol usage, though!
